### PR TITLE
Parse objects from config itself with as[]

### PR DESCRIPTION
**Pull request by Tolsi**
I saw another pull request #51 too late and I offer a less flexible, but simpler solution to the problem #8

### CODE REVIEW

File: src/main/scala/net/ceedubs/ficus/FicusConfig.scala [hunk 1]
```diff
@@ -9,6 +9,8 @@ trait FicusConfig {
 
   def as[A](path: String)(implicit reader: ValueReader[A]): A = reader.read(config, path)
 
+  def as[A](implicit reader: ValueReader[A]): A = reader.read(config, ".")
```
Comment (kailuowang) [inline src/main/scala/net/ceedubs/ficus/FicusConfig.scala]: this is a nit, how about we make it
```scala
def as[A](implicit reader: ValueReader[A]): A = as(".")
```
The intention here is to expose the fact that if you pass in a "." as `path` you get the same result as `as`.

**Comment by kailuowang** [comment]
Thanks! I think this would work. I left a minor comment.

**Comment by Tolsi** [comment]
@kailuowang done!

**Comment by kailuowang** [comment]
1.4.3 released

**Comment by Tolsi** [comment]
Cool!

**Pull request merged**